src/rust.rs

```diff
@@ -400,11 +400,11 @@ macro_rules! rooted {
     };
     (in($cx:expr) let $name:ident: $type:ty) => {
         let mut __root = $crate::jsapi::Rooted::new_unrooted();
-        let $name = $crate::rust::RootedGuard::new($cx, &mut __root, $type::default());
+        let $name = $crate::rust::RootedGuard::new($cx, &mut __root, <$type as GCMethods>::initial());
```

Comment:
These will need to use `$crate::rust::GCMethods` to avoid causing errors like the ones on TravisCI, I believe.

Reply:
@jdm Test error has been resolved.